### DISCUSSION TITLE
How are you getting HA to expose folders other than /www?

### DISCUSSION THREAD

**Discussion by nappyjim** [moralmunky/Home-Assistant-Mail-And-Packages#859] [Q&A]
So I am trying to create a notification that goes out to my phone (android) when an Amazon package has been delivered.  But I can not seem to get HA to expose any directory other than the /www one.

To test, I had an image called 1.jpg, I copied this image to /www folder.  When I go to my external URL provided by Naba Casa  + /local/1.jpg, I see the image.  Talking to the android app dev of HA, if I can see it using my external URL, I will be able to display the image in the notification.

I take that same image, 1.jpg and copy it to /config/custom_components/mail_and_packages/images/amazon/ and when I try to view it using the same method as above, external URL + /config/custom_components/mail_and_packages/images/amazon/, I do NOT see the image.

So my question is, how are you guys attaching delivery images to your notifications if those images are in the /www folder.

**Comment by firstof9** [comment]
Use the camera entity.
